.ci_support/linux_64_llvm14llvmdev14.yaml

```diff
@@ -15,19 +15,23 @@ channel_targets:
 cxx_compiler:
 - gxx
 cxx_compiler_version:
-- '12'
+- '13'
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+fmt:
+- '10'
 gmp:
 - '6'
 libboost_devel:
-- '1.84'
+- '1.86'
 llvm:
 - '14'
 llvmdev:
 - '14'
 mpfr:
 - '4'
+spdlog:
+- '1.13'
```

Comment:
> but I am having some troubles understanding why `spdlog=1.14` is not being considered by the solver.

Because it's pinned to 1.13, unless you include https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/spdlog114.yaml under `.ci_support/migrations`

Reply:
Should I do this manually now or better to wait for the bot to open a PR?

Reply:
The bot cannot open a PR because the current state on the main branch is pinned
https://github.com/conda-forge/heyoka-feedstock/blob/854f19399556af9907c4e8877e1bc687bd0761b4/recipe/meta.yaml#L53

It's probably easiest to just to it manually in this case. Alternative would be to merge a PR that unpins spdlog/fmt, wait for the bot to issue a PR, then restart/rebase this one.

Reply:
Cheers, I just did that.

spdlog is pinned on the main branch because at one point I could not figure out how to get the correct (i.e., compatible) combination of spdlog/fmt versions in the recipe.

Reply:
Well this is looking messier than I thought... I suppose I should also add the fmt migration, but the problem is that the current version of heyoka does not support fmt 11 because of API changes in fmt.

I was about to release a new version of heyoka which does support fmt 11, and I am inclined to think it would be better to re-run the migration on the new version rather than trying to make this work.

Reply:
Yeah, you need to support fmt11 for boost 1.86 in any case... (same situation as for obake)